packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -281,6 +284,13 @@ export default function Sidebar({
     contirbuteMenuItems.push(submitArticleMenuItem);
   }
 
+  contirbuteMenuItems.push({
```

Comment:
This is probably my bad but it should be 
```suggestion
  contributeMenuItems.push({
```